From 294f26f451ac258cb808c01807ec53e20c879f50 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Tuomas=20R=C3=A4s=C3=A4nen?= <tuomas.rasanen@opinsys.fi>
Date: Tue, 20 Aug 2024 19:28:32 +0300
Subject: [PATCH 13/14] [puavo] appearance: fix dark suffix strip order

New order ensures longest possible suffix gets stripped, e.g. dash is
not left hanging around.

Example:

Before: Yaru-dark -> Yaru-
After: Yaru-dark -> Yaru
---
 panels/background/cc-background-panel.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/panels/background/cc-background-panel.c b/panels/background/cc-background-panel.c
index 9aa6fb710..e1ca58481 100644
--- a/panels/background/cc-background-panel.c
+++ b/panels/background/cc-background-panel.c
@@ -80,7 +80,7 @@ struct _CcBackgroundPanel
   CcIconThemeChooser *icon_theme_chooser;
 };
 
-static const gchar *const DARK_SUFFIXES[] = {"Dark", "dark", "-Dark", "-dark"};
+static const gchar *const DARK_SUFFIXES[] = {"-Dark", "-dark", "Dark", "dark"};
 static const int DARK_SUFFIX_COUNT = sizeof(DARK_SUFFIXES) / sizeof(DARK_SUFFIXES[0]);
 
 CC_PANEL_REGISTER (CcBackgroundPanel, cc_background_panel)
-- 
2.39.2

